### PR TITLE
[apps/graph] feat(graph): extending type Query. Add embedded postgress to work with this locally

### PR DESCRIPTION
**Pull request by alber70g**
- feat(graph): adds Query field blocks
- chore(graph-client): use fragments for block fields. Change interface for useParsedBlocks
- chore(graph): moves prismaClient to db. Adds embedded postgres and worker for development

### CODE REVIEW

File: packages/apps/graph/package.json [hunk 1]
```diff
@@ -29,10 +29,12 @@
   },
   "dependencies": {
     "@pothos/core": "~3.24.0",
+    "@pothos/plugin-mocks": "~3.7.7",
```
Comment (alber70g) [inline packages/apps/graph/package.json:32]: I've removed this dependency

File: packages/apps/graph/src/utils/dev-database/startEmbeddedPostgres.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+const REGEX_STATEMENT_INSERT_INTO: RegExp =
+  /"statement: INSERT INTO.*?(?<!")",/;
+const REGEX_STATEMENT_BEGIN: RegExp = /"statement: BEGIN.*?(?<!")",/;
+const REGEX_STATEMENT_COMMIT: RegExp = /"statement: COMMIT.*?(?<!")",/;
+
+// eslint-disable-next-line @rushstack/typedef-var
+const PRISMA_ENV_OPTIONS = {
+  DATABASE_URL:
+    'postgresql://graphuser:graphpassword@localhost:55432/chainweb-data',
+} as const;
```
Comment (alber70g) [inline packages/apps/graph/src/utils/dev-database/startEmbeddedPostgres.ts:19]: Use env var

File: packages/apps/graph/src/schema.graphql [hunk 3]
```diff
@@ -280,13 +280,38 @@ enum KeysetPredicate {
...
-  newBlocks: [Block!]!
+  newBlocks(filter: NewBlocksFilterInput): [Block!]!
   events(filter: EventsFilterInput): [TransactionEvent!]!
   poll(requestKey: String!): String!
   # todo
   # send(sendBody: SendBody): SendResponse
 }
 
+input NewBlocksFilterInput {
+  chains: [Int!]! = [
```
Comment (jermaine150) [inline packages/apps/graph/src/schema.graphql:291]: Shouldn't the chains run from 0 to 19?

Comment (alber70g) [reply packages/apps/graph/src/schema.graphql:291]: fixed

**Review by jermaine150 — APPROVED** [review]

**Pull request merged**